File: app/src/main/java/com/edanichev/nounIcons/app/main/NounHintCloud/View/HintCloudViewInterface.java
package com.edanichev.nounIcons.app.main.NounHintCloud.View;

import com.arellomobile.mvp.MvpView;
import com.edanichev.nounIcons.app.main.NounHintCloud.Model.CloudTagsModel;

public interface HintCloudViewInterface extends MvpView {
    void showHintCloud();

    void hideHintCloud();

    void addChipsToHintCloud(CloudTagsModel tags);

    void initView();
}

File: app/src/main/java/com/edanichev/nounIcons/app/main/NounHintCloud/View/HintCloudViewInterface.kt
package com.edanichev.nounIcons.app.main.NounHintCloud.View

import com.edanichev.nounIcons.app.main.NounHintCloud.Model.CloudTagsModel
import com.edanichev.nounIcons.app.main.NounHintCloud.Presenter.HintCloudPresenterInterface
import com.edanichev.nounIcons.app.main.base.data.BaseView

interface HintCloudViewInterface : BaseView<HintCloudPresenterInterface> {

    fun showHintCloud()

    fun hideHintCloud()

    fun addChipsToHintCloud(tags: CloudTagsModel)

    fun initChips()
}